### PR TITLE
fix : possible JValue element causes exception.

### PR DESCRIPTION
**Pull request by Deren-Liao**
https://github.com/GoogleCloudPlatform/google-cloud-visualstudio/issues/566

### CODE REVIEW

**Comment by ILMTitan** [comment]
I don't understand what this is trying to fix. Is there a case that wasn't JArray, JObject, JProperty or JValue? Was there a null pointer at `jItem.Value.GetType().Name`?

File: GoogleCloudExtension/GoogleCloudExtension/StackdriverLogsViewer/TreeViewConverters/JObjectNode.cs [hunk 1]
```diff
@@ -40,16 +41,13 @@ protected override void ParseObjectTree(object obj)
             JObject jsonObj = obj as JObject;
             foreach (var jItem in jsonObj)
             {
-                switch (jItem.Value.GetType().Name)
+                if (jItem.Value is JArray || jItem.Value is JObject)
```
Comment (Deren-Liao) [inline GoogleCloudExtension/GoogleCloudExtension/StackdriverLogsViewer/TreeViewConverters/JObjectNode.cs]: This is code refactoring.   Not a bug.

File: GoogleCloudExtension/GoogleCloudExtension/StackdriverLogsViewer/TreeViewConverters/JArrayNode.cs [hunk 2]
```diff
@@ -38,7 +40,19 @@ public JArrayNode(string name, JArray jArrayObj, ObjectNodeTree parent)
         protected override void ParseObjectTree(object obj)
         {
             JArray jsonArray = obj as JArray;
-            ParseArray(jsonArray.Children());
+            List<object> objectArray = new List<object>();
```
Comment (Deren-Liao) [inline GoogleCloudExtension/GoogleCloudExtension/StackdriverLogsViewer/TreeViewConverters/JArrayNode.cs]: This is the fix.  If element is JValue,  the ParseArray would throw exception.  
https://github.com/GoogleCloudPlatform/google-cloud-visualstudio/blob/master/GoogleCloudExtension/GoogleCloudExtension/StackdriverLogsViewer/TreeViewConverters/ObjectNodeTree.cs#L178
And then
https://github.com/GoogleCloudPlatform/google-cloud-visualstudio/blob/master/GoogleCloudExtension/GoogleCloudExtension/StackdriverLogsViewer/TreeViewConverters/ObjectNodeTree.cs#L143

**Comment by Deren-Liao** [comment]
Added comment, let me know if it's still not clear.

**Comment by Deren-Liao** [comment]
Thank you.

**Pull request merged**